src/main.rs

```diff
@@ -50,6 +51,17 @@ enum Opt {
         )]
         sort: SortOrder,
 
+        /// Analyze .ll files that were produced by eg. `RUSTFLAGS="--emit=llvm-ir" ./x.py build --stage 0 compiler/rustc`.
+        /// Supports globs, eg `./build/x86_64-unknown-linux-gnu/stage0-rustc/x86_64-unknown-linux-gnu/debug/deps/*.ll`
+        // zsh substitutes the glob with a list of files, therefore this needs to be a Vec.
```

Comment:
Is there a reason to both support globs and multiple files? It seems easier to let the shell expand the glob instead (and means less dependencies for llvm-lines.

Reply:
Changed in #33 to leave globbing to the shell.